packages/load-script/test/callback-handler.js

```diff
@@ -213,25 +212,4 @@ describe( 'loadScript/callback-handler', () => {
 			expect( elementObject.onload ).toBeNull();
 		} );
 	} );
-
-	describe( 'handleRequestError( event )', () => {
-		const url = '/';
-		const elementObject = { getAttribute: () => url };
-		const eventObject = { target: elementObject };
-		const callback = jest.fn();
-
-		beforeAll( function() {
-			addScriptCallback( url, callback );
-			handleRequestError.bind( elementObject )( eventObject );
-		} );
-
-		test( 'should execute callbacks associated with the url with an error', () => {
-			expect( callback ).toHaveBeenCalledTimes( 1 );
-			expect( callback ).toHaveBeenCalledWith( new Error( `Failed to load script "${ url }"` ) );
-		} );
-
-		test( 'should set this.onerror to null', () => {
-			expect( elementObject.onerror ).toBeNull();
-		} );
-	} );
```

Comment:
This behavior changed with the `onload`/`onerror` -> `addEventListener` change.

We may be able to get similar tests if they're valuable, but migrating these existing tests as they are proved very complicated. I opted to remove them but I'm open to reimplementing similar tests.